internal/kafka/internal/routes/route_loader.go

```diff
@@ -212,6 +214,15 @@ func (s *options) buildApiBaseRouter(mainRouter *mux.Router, basePath string, op
 
 	apiV1Router.HandleFunc("", v1Metadata.ServeHTTP).Methods(http.MethodGet)
 
+	// /api/kafkas_mgmt/v1/instance_types/{cloud_provider}/{cloud_region}
+	apiV1SupportedKafkaInstanceTypesRouter := apiV1Router.PathPrefix("/instance_types/{cloud_provider}/{cloud_region}").Subrouter()
+	apiV1SupportedKafkaInstanceTypesRouter.HandleFunc("", supportedKafkaInstanceTypesHandler.ListSupportedKafkaInstanceTypes).
+		Name(logger.NewLogEvent("list-supported-kafka-instance-types-by cloud-region", "list supported kafka instance types by cloud region").ToString()).
+		Methods(http.MethodGet)
+	apiV1SupportedKafkaInstanceTypesRouter.Use(requireIssuer)
+	apiV1SupportedKafkaInstanceTypesRouter.Use(requireOrgID)
+	apiV1SupportedKafkaInstanceTypesRouter.Use(authorizeMiddleware)
```

Comment:
Hmm, since the cloud provider endpoints for getting the cloud provider id and regions also require authentication, we may need to have it here as well. Can ignore this comment :)

Reply:
Yeah there's a comment in the Jira confirming this